### PR TITLE
Add ESLint

### PR DESCRIPTION
**Pull request by MrBurrBurr**

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,36 +1,36 @@
 <template>
 	<div class="app" :class="[{ 'app--not-authorized': !authenticated, 'app--small-navigation': smallNavigation, 'app--boxed-layout': boxedLayout, 'app--dark-mode': darkMode }, themeClass]">
-		<app-header></app-header>
-		<app-navigation></app-navigation>
-		<app-side-menu></app-side-menu>
+		<app-header />
```
Comment (Aareksio) [inline src/App.vue]: Disable that rule please

Comment (MrBurrBurr) [reply src/App.vue]: Sure thing

**Comment by JustArchi** [comment]
You most likely want to check the issue with package-lock.json indentation, unless you want to fight with dependabot every time it decides to update a dependency.

**Comment by MrBurrBurr** [comment]
Yep, that was annoying... but I think it should be good once this PR is merged since dependabot wont bother changing indents.

**Comment by JustArchi** [comment]
I don't know, it probably uses `npm` to update it, if npm is smart enough to respect current indent or editorconfig then probably yes.

**Comment by MrBurrBurr** [comment]
Lets find out together :D

**Pull request merged**